index.js

```diff
@@ -1012,9 +1043,45 @@ class GithubScm extends Scm {
                 username: hoek.reach(webhookPayload, 'sender.login'),
                 lastCommitMessage: hoek.reach(webhookPayload, 'head_commit.message') || '',
                 hookId,
-                scmContext: scmContexts[0]
+                scmContext: scmContexts[0],
+                ref: hoek.reach(webhookPayload, 'ref')
             };
+        case 'release': {
+            return {
+                action: 'release',
+                branch: hoek.reach(webhookPayload, 'release.target_commitish'),
+                checkoutUrl,
+                type: 'repo',
+                username: hoek.reach(webhookPayload, 'sender.login'),
+                hookId,
+                scmContext: scmContexts[0],
+                ref: hoek.reach(webhookPayload, 'release.tag_name')
+            };
+        }
+        case 'create': {
+            const refType = hoek.reach(webhookPayload, 'ref_type');
+
+            if (refType !== 'tag') {
+                winston.info('%s event of %s is not available yet in scm-github plugin', type, refType);
+
+                return null;
+            }
+
+            return {
+                action: 'tag',
+                branch: hoek.reach(webhookPayload, 'master_branch'),
+                checkoutUrl,
+                type: 'repo',
+                username: hoek.reach(webhookPayload, 'sender.login'),
+                hookId,
+                scmContext: scmContexts[0],
+                ref: hoek.reach(webhookPayload, 'ref')
+            };
+        }
+
         default:
+            winston.info('%s event is not parsable.', type);
```

Comment:
Same here. `%s event is not available yet in scm-github plugin`

Reply:
fixed!